bootstrap.go

```diff
@@ -53,6 +53,7 @@ func Start(ctx context.Context, opts ...Option) (StopFunc, error) {
 	}
 
 	err := internal.VerifyEnvVarsSet(
+		internal.DatadogAgentHost,
```

Comment:
This does not look right to me. You are now requiring that `DatadogAPMEndpoint` (`DD_TRACE_AGENT_URL`) AND `DatadogAgentHost` (`DD_AGENT_HOST`) must be set!

But the logic in the dd-trace-go library [is the following](https://github.com/DataDog/dd-trace-go/blob/main/internal/agent.go#L41): 
```
// AgentURLFromEnv resolves the URL for the trace agent based on
// the default host/port and UDS path, and via standard environment variables.
// AgentURLFromEnv has the following priority order:
//   - First, DD_TRACE_AGENT_URL if it is set
//   - Then, if either of DD_AGENT_HOST and DD_TRACE_AGENT_PORT are set,
//     use http://DD_AGENT_HOST:DD_TRACE_AGENT_PORT,
//     defaulting to localhost and 8126, respectively
//   - Then, DefaultTraceAgentUDSPath, if the path exists
//   - Finally, localhost:8126
```

So it makes more sense that we check for **at least one** of  `DD_TRACE_AGENT_URL` or `DD_AGENT_HOST` to be set.

Reply:
`DD_AGENT_HOST` will not be used if `DD_TRACE_AGENT_URL` is set.

Reply:
@AtzeDeVries can we use `DD_TRACE_AGENT_URL` for TCP?

Reply:
Previously, we used `tracer.WithUDS` and `tracer.WithAgentAddr` (https://github.com/coopnorge/go-datadog-lib/blob/main/legacy_bootstrap.go#L122-L129), which may have made it difficult.

We must add a normalization to allow `/var/run/datadog/apm.socket` to be accepted.
`url.Parse("/var/run/datadog/apm.socket")` yields an empty scheme.

Reply:
@AtzeDeVries , would #453 remove the need to set `DD_AGENT_HOST`?